pkg/bmc/bmc.go

```diff
@@ -978,7 +947,7 @@ func redfishConnect(
 		Insecure: true,
 	}
 
-	ctx, cancel := context.WithTimeout(context.TODO(), sessionTimeout)
+	ctx, cancel := context.WithTimeout(context.Background(), sessionTimeout)
```

Comment:
a bit of a nit, but we've adopted `context.TODO()` as a standard for this repo, could you remove this change please?

Reply:
Fixed